src/main/kotlin/pl/edu/agh/gem/internal/model/Group.kt

```diff
@@ -0,0 +1,25 @@
+package pl.edu.agh.gem.internal.model
+
+data class Group(
+    val id: String,
+    val name: String,
+    val color: Long,
+    val ownerId: String,
+    val members: Members,
+    val acceptRequired: Boolean,
+    val baseCurrency: Currencies,
```

Comment:
according to out last talk it should be renamed to groupCurrency or just currency.